### DISCUSSION TITLE
Ref Arch permissions error between app accound and logs

### DISCUSSION THREAD

**Discussion by pete0emerson** [gruntwork-io/knowledge-base#281] [Help]
Hello, I'm seeing a permissions error for the RefArch deployment for the SNS:Publish actions in our alarms, between an app account group and the logs account. How do I fix this? Here's an example:
```Failed to execute action arn:aws:sns:us-west-2:000000000000:BenchmarkAlarmTopic. Received error: "Resource: arn:aws:cloudwatch:us-west-2:111111111111:alarm:SignInWithoutMFA is not authorized to perform: SNS:Publish on resource: arn:aws:sns:us-west-2:000000000000:BenchmarkAlarmTopic because no resource-based policy allows the SNS:Publish action"```

**Comment by Etiene** [comment]
Hi @pete0emerson, is this maybe related to this other question? https://github.com/gruntwork-io/knowledge-base/discussions/139

**Reply by antonl** [reply]
I think this is different. In my case, the metrics are actually computed in the same account and it is the SNS publish action to the logging account that fails. This is probably a bug with the Reference Architecture.
